src/openagi/llms/claude.py

```diff
@@ -0,0 +1,62 @@
+from openagi.llms.base import LLMBaseModel, LLMConfigModel
+from openagi.utils.yamlParse import read_from_env
+from typing import Any
+
+from langchain_core.messages import HumanMessage
+
+from openagi.exception import OpenAGIException
+try:
+   from langchain_anthropic import ChatAnthropic
+except ImportError:
+  raise OpenAGIException("Install Langchain Anthropic to use Claude LLM `pip install langchain-anthropic`")
+
+class ChatAnthropicConfigModel(LLMConfigModel):
+    """
+    Configuration model Anthropic model. This provides opus, sonnet SOTA models
+    """
+    anthropic_api_key: str
+    model_name: str = "claude-3-opus-20240229"
```

Comment:
Claude supports 3.5 sonnet to public. The base model can be claude-3-5-sonnet-20240620 instead of opus